numba/core/ir_utils.py

```diff
@@ -1737,6 +1737,12 @@ def dump_blocks(blocks):
         for stmt in block.body:
             print("    ", stmt)
 
+def is_operator(expr):
```

Comment:
This name seems a bit strange as `*getitem` is in there? How about `is_operator_or_getitem`?